bin/upload.js

```diff
@@ -0,0 +1,29 @@
+/*
+  An S3 uploader with some useful default settings for Zooniverse static files.
+  Accepts an S3 path, relative to zooniverse-static, as its only argument.
+  Usage example: node upload.js pandora.zooniverse.org
+*/
+const Uploader = require('s3-batch-upload').default;
+
+const args = process.argv.slice(2);
+const remotePath = args[args.length - 1];
+
+async function uploadBuild() {
+  const uploader = new Uploader({
+    bucket: 'zooniverse-static',
+    localPath: './dist',
+    remotePath,
+    concurrency: '200',
+    cacheControl: {
+      '**/index.html': 'no-cache, must-revalidate',
+      '**/*.*': 'max-age=86400'
```

Comment:
Should these assets be longer in the cache and immutable. Each one should be uniquely identifiable and immutable to change i believe if our build tools add modified paths if the underlying asset content changes...Thoughts? 
```suggestion
      '**/*.*': 'public, max-age=604800, immutable'
```
based on `Caching static assets` in https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Cache-Control

Reply:
Sounds good to me. From reading the [MDN page](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Cache-Control), most browsers don't use `immutable` yet but I don't see any harm in adding it.